ui/app/public/index.html

```diff
@@ -9,10 +9,13 @@
     <!-- Global site tag (gtag.js) - Google Analytics -->
     <script async src="https://www.googletagmanager.com/gtag/js?id=G-MSGFSQQEJ3"></script>
     <script>
-      window.dataLayer = window.dataLayer || [];
-      function gtag(){dataLayer.push(arguments);}
-      gtag('js', new Date());
-      gtag('config', 'G-MSGFSQQEJ3');
+      if (window.location.href === "prod.sifchain.finance") {
```

Comment:
Should this be `dex.sifchain.finance`?

Reply:
Indeed it should, one of those things I triple checked and still got wrong lol one moment